xbmc/AppParamParser.cpp

```diff
@@ -21,18 +21,33 @@
 #include <string>
 #include <vector>
 
-#if defined(TARGET_LINUX)
 namespace
 {
-std::vector<std::string> availableWindowSystems = CCompileInfo::GetAvailableWindowSystems();
-std::array<std::string, 1> availableLogTargets = {"console"};
+
+constexpr const char* versionText =
+    R"""({0} Media Center {1}
+Copyright (C) {2} Team {0} - http://kodi.tv
```

Comment:
How serious are we taking rebranding? There any other way to set the url other than a hardcoded string?

Reply:
Yea I'm not sure if this was left or just missed when the rebranding stuff happened.